tests/unit/commands/local/lib/test_sam_function_provider.py

```diff
@@ -1780,6 +1782,23 @@ def test_must_skip_non_existent_properties(self):
         self.assertEqual(expected, result)
 
 
+class TestSamFunctionProvider_build_function_configuration(TestCase):
+    def test_must_convert(self):
```

Comment:
Think this test name should be descriptive like test_handler_missing_exception

Reply:
Could also consider making another test for the emtpy handler string for maximum coverage

Reply:
ah yea i forgot to change the name from the test that i copied